src/Graphics/AnimationFlashing.h

```diff
@@ -2,15 +2,19 @@
 #define _ANIMATION_FLASHING_H_
 
 #include "Animation.h"
+#include "SharedProperty.h"
 
-class AnimationFlashing : public Animation {
-	public:
-		AnimationFlashing(Shape* shape, int64_t interval) : Animation(shape), _interval(interval) { }
-		void run(int64_t millis);
-		
-	private:
-		int64_t _interval;
-		bool _isDrawn = false;
+class AnimationFlashing : public Animation
+{
+public:
```

Comment:
We're passing in a pointer to AnimationFlashing to a blink sharedproperty, so AnimationFlashing can use it to change the blink status